### PR TITLE
[velero] Bump up Velero 1.14.0

### PR DESCRIPTION
**Pull request by xebia-progress**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped, please refer to the [chart version instruction](https://github.com/vmware-tanzu/helm-charts/blob/main/RELEASE-INSTRUCT.md#guidelines)
- [x] Variables are documented in the values.yaml or README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

**Comment by xebia-progress** [comment]
@jenting That's done.

File: charts/velero/README.md [hunk 1]
```diff
@@ -93,12 +93,14 @@ helm upgrade vmware-tanzu/velero <RELEASE NAME> --reuse-values --set configurati
 ```
 ## Upgrading Chart
 
-### Upgrading to 6.0.0
-
-This version removes the `nodeAgent.privileged` field, you should use `nodeAgent.containerSecurityContext.privileged` instead
```
Comment (jenting) [inline charts/velero/README.md:98]: We should keep it as is.

**Review by jenting — COMMENTED** [review]
Could you please check the CI failure?
Looks like the CRD is not correct with v1.14 version.

File: charts/velero/README.md (hunk 1 above)
Comment (xebia-progress) [reply charts/velero/README.md]: Restored

**Comment by xebia-progress** [comment]
@jenting The CRDs are updated now.

File: charts/velero/README.md [hunk 2]
```diff
@@ -93,12 +93,18 @@ helm upgrade vmware-tanzu/velero <RELEASE NAME> --reuse-values --set configurati
 ```
 ## Upgrading Chart
 
+### Upgrading to 7.0.0
```
Comment (jenting) [inline charts/velero/README.md:96]: Do we have special notes such as not backward compatible compare to the previous version? If no, we could remove this one.

File: charts/velero/values.yaml [hunk 3]
```diff
@@ -105,14 +105,8 @@ dnsPolicy: ClusterFirst
 # Init containers to add to the Velero deployment's pod spec. At least one plugin provider image is required.
 # If the value is a string then it is evaluated as a template.
 initContainers:
-  # - name: velero-plugin-for-csi
```
Comment (jenting) [inline charts/velero/values.yaml:108]: Any reasons on why remove this?

**Review by jenting — APPROVED** [review]
Awesome work 👏 
LGTM!

**Review by blackpiglet — APPROVED** [review]

**Comment by NoamGaloz1** [comment]
@reasonerjt

**Pull request merged**